proposals/mdawson-cpc-charter/CPC-Charter.md

```diff
@@ -0,0 +1,305 @@
+
+ * The TOC charter, model, approach, composition, etc may be modified by a two-thirds vote of the entire Governing Board.
+
+===========
+
+Changes to CPC membership should be posted in the agenda, and may be suggested
+as any other agenda item.
+
+No more than one-fourth of the CPC members may be affiliated with the
+same employer. If removal or resignation of a CPC member, or a change of
+employment by a CPC member, creates a situation where more than
+one-fourth of the CPC membership shares an employer, then the situation
+must be immediately remedied by the resignation or removal of one or more
+PC
+
+The CPC may, at its discretion, invite any number of non-voting observers to
+participate in the public portion of CPC discussions and meetings.
+
+The CPC shall meet regularly using tools that enable participation by the
+community. The meeting shall be directed by
+the CPC Chairperson. Responsibility for directing individual meetings may be
+delegated by the CPC Chairperson to any other CPC member. Minutes or an
+appropriate recording shall be taken and made available to the community
+through accessible public postings.
+
+\>\>\>\>\>\>\>\>\>\>\> Node.js
+
+CPC members are expected to regularly participate in CPC activities.
+In the case where an individual CPC member -- within any three month period --
+attends fewer than 25% of the regularly scheduled meetings, does not
+participate in CPC discussions, *and* does not participate in CPC votes, the
+member shall be automatically removed from the CPC. The member may be invited
+to continue attending CPC meetings as an observer.
+
+===========
+
+
+Section 5. Responsibilities and Expectations of the CPC
+
+Subject to such policies as may be set by the Board, the CPC is responsible for
+ensuring: 
+
+   * ensuring collaboration is the driving principle within a Project, between
+     _$FOUNDATION_ Projects, and between _$FOUNDATION_ Projects and the broader
+     community.
+   * defining and maintaining neutral consensus for the technical vision for the _$FOUNDATION_
+   * approving new projects within the scope for _$FOUNDATION_ set by the Governing Board,
+     as outlined in [XXXX], and creating conceptual architecture for how the proejcts
+     fit together.
+   * aligning projects, removing or archiving projects, as outlined in [XXXX].
+   * defining common practices to be implemented across _$FOUNDATION_ projects, if any.
+   * The CPC is also responsible for mediating technical conflicts between
+     _$FOUNDATION_ Projects when attempts to resolve those conflicts within the Project
+    and were unsuccessful. 
+   * The CPC serves as the _$FOUNDATION_'s primary technical
+     liaison body with external open source projects, consortiums and groups and is
+     also responsible for technical participation in standards bodies
```

Comment:
This has a slight implication that, in order to participate as $FOUNDATION to standard bodies, you have to be a member of CPC. I think that would be a bit elitist. IMHO somebody might be interested in being part of the CPC but not participate in standards and vice versa.

I think this responsibility should be delegated to a subcommitee/or somehow worded differently to enable such delegation.

Reply:
I would read it more that the CPC has the responsibility to make sure that the right things happen, not that you have to be on the CPC itself.

Reply:
Then we should probably align the test, this seems to be a direct responsibility.

Reply:
Updated the text, let me know if that sounds better.